src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/OptionMetadata.php

```diff
@@ -81,15 +81,15 @@ public function getType()
     }
 
     /**
-     * @return int|string|OptionMetadata[]
+     * @return bool|int|string|float|OptionMetadata[]|mixed|null
```

Comment:
same as above no float, no mixed

```suggestion
     * @return bool|int|string|null|OptionMetadata[]
```